TTS/tts/layers/xtts/tokenizer.py

```diff
@@ -535,11 +536,50 @@ def korean_cleaners(text):
 class VoiceBpeTokenizer:
     def __init__(self, vocab_file=None):
         self.tokenizer = None
-        self.katsu = None
         if vocab_file is not None:
             self.tokenizer = Tokenizer.from_file(vocab_file)
+        self.char_limits = {
+            "en": 250,
+            "de": 253,
+            "fr": 273,
+            "es": 239,
+            "it": 213,
+            "pt": 203,
+            "pl": 224,
+            "zh-cn": 82,
+            "ar": 166,
+            "cs": 186,
+            "ru": 182,
+            "nl": 251,
+            "tr": 226,
+            "ja": 71,
+            "hu": 224,
+            "ko": 95,
+        }
```

Comment:
Maybe move these right under `class VoiceBpeTokenizer:`? That'd make each instance share the same dict. (Also, should these be sorted?)

Reply:
Should only be one instance at all times.